src/components/Storefront/Footer.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { Image, Center, Text, Link, Button, HStack, VStack, Input } from '@chakra-ui/react';
+import { ChevronRightIcon } from '@chakra-ui/icons';
+import { FaFacebookSquare, FaInstagramSquare, FaYoutube, FaLinkedin, FaTwitter } from 'react-icons/fa';
+import smallLogo from '../../assets/building_up_footer_small_icon.png';
+
+const Footer = props => {
+  const { subscribtionEmail, changeSubscriptionEmail } = useState();
+
+  const subscribe = e => {
+    e.preventDefault();
+    changeSubscriptionEmail(e);
+  };
+
+  const updateSubscriptionEmail = e => {
+    e.preventDefault();
+    alert(subscribtionEmail);
+  };
+
+  return (
+    <VStack spacing="0px">
+      <HStack backgroundColor="black" w="100%" display="flex" paddingTop="54px" paddingBottom="84px">
+        <Image src={smallLogo} alt="Small Icon" marginLeft="41px" marginRight="47px" alignSelf="flex-start" />
+        <HStack backgroundColor="black" w="100%" display="flex" justifyContent="space-between">
+          <VStack w="13%" alignItems="flex-start" spacing="26px">
+            <Text fontWeight="extrabold" letterSpacing="wider" color="white">
+              GET IN TOUCH
+            </Text>
+            <Text color="white" fontFamily="Karla">
+              116 Industry Street <br /> Suite 313 <br /> Toronto, ON M6M 4L8
+            </Text>
+            <Link href="mailto:info@raisingtheroof.org" color="white" fontFamily="Karla" fontWeight="normal">
+              info@raisingtheroof.org
+            </Link>
+            <HStack spacing="5">
+              <FaFacebookSquare fontSize="26px" color="white" />
+              <FaInstagramSquare fontSize="26px" color="white" />
+              <Center h="22px" w="22px" backgroundColor="white" borderRadius="2px">
+                <FaYoutube />
+              </Center>
+              <FaLinkedin fontSize="26px" color="white" />
+              <Center h="22px" w="22px" backgroundColor="white" borderRadius="2px">
+                <FaTwitter />
+              </Center>
```

Comment:
do you know why these two have to be styled differently from the ones above?

Reply:
So for those 2, the designer chose to not use the default square icon, but instead the no-square icon and put a box around withy with rounded edges.

Reply:
let's just use the square-icon for all for consistency